setup.py

```diff
@@ -13,7 +13,7 @@
 
 requires = [
     'pyramid>=1.3',
-    'six', # required by `ldappool` but not in their requirements file
+    'six',  # required by `ldappool` but not in their requirements file
```

Comment:
If this is no longer required because `ldappool` itself is Py3 only and no longer uses `six` then let's remove this.